src/pages/docs/reference/fulfillment-delegation/models/carrier/addresses/address-base.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: AddressBase
+description: Basic structure for an address
+---
+
+AdvancedOptions
```

Comment:
This should be AddressBase

Reply:
Corrected this and a few other copy/paste typos